.build/parent-pom-template.xml

```diff
@@ -1029,6 +1024,28 @@
         <artifactId>agrona</artifactId>
         <version>1.17.1</version>
       </dependency>
+      <dependency>
+        <groupId>org.apache.lucene</groupId>
+        <artifactId>lucene-core</artifactId>
+        <version>7.5.0</version>
+      </dependency>
+      <dependency>
+        <groupId>org.apache.lucene</groupId>
+        <artifactId>lucene-analyzers-common</artifactId>
```

Comment:
I guess we can remove the analyzers bit now?

Reply:
No, I have reverted the change in SASI for this patch so this can go.